scripts/generators/ecs_helpers.py

```diff
@@ -6,11 +6,11 @@
 # Dictionary helpers
 
 
-def dict_copy_keys_ordered(dict, copied_keys):
+def dict_copy_keys_ordered(dct, copied_keys):
```

Comment:
I renamed this to avoid variable shadowing, since `dict` already means something.
It wasn't strictly necessary, but it makes the code easier to read, and makes PyCharm more pleased with me.